FAQs/README.md

```diff
@@ -338,3 +341,50 @@ stages:
         azurePowerShellVersion: 'LatestVersion'
 ```
 
+- ### **UI**
+
+### **1. I am not able to see subscriptions in AzTS UI. Are there any prequisites or any settings that I am missing?**
+If you are not able to see subscriptions in AzTS UI , please check below prerequisites followed by troubleshooting steps:
+
+### Prerequisites:
+1. Signed in user must have one of the following permissions at subscription or resource group scope: Owner, Contributor, ServiceAdministrator, CoAdministrator, AccountAdministrator, Security Reader, Security Admin.
+
+2. Subscription scan should have completed for the day. Automated AzTS scans are configured to start at approximately 1:00 AM UTC. You can use the [On-Demand scan](../01-Setup%20and%20getting%20started/README.md#2-manually-trigger-azts-on-demand-scan-for-entire-tenant) command to trigger the scan immediately after the installation.
+
+3. In case access is recently granted, it will reflect in AzTS from next scan onwards. So in case of automated AzTS scan it would take next 24 hours to reflect latest access. You can use the [On-Demand scan](README.md#2-manually-trigger-azts-on-demand-scan-for-entire-tenant) command to trigger the scan in case you want to reflect newly added access immediately.
```

Comment:
If access to resources has been recently granted, it may take up to 24 hours for the changes to be reflected in AzTS during the next automated scan. If you would like to see the newly added access reflected immediately, you can use the on-demand scan command to trigger a scan outside of the regular schedule. This will ensure that the latest access information is updated in AzTS without waiting for the next automated scan.

Reply:
Updated